editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -2808,11 +2812,14 @@ void TileSetEditor::draw_polygon_shapes() {
 							colors.push_back(c_bg);
 						}
 					} else {
-						PoolVector<Vector2> vertices = shape->get_vertices();
-						for (int j = 0; j < shape->get_polygon(0).size(); j++) {
-							polygon.push_back(vertices[shape->get_polygon(0)[j]] + anchor);
-							colors.push_back(c_bg);
+						for (int outline_inx = 0; outline_inx < shape->get_outline_count(); outline_inx++) {
```

Comment:
We use `idx` throughout the codebase whenever an abbreviation for `index` is needed.